coc/abc.py

```diff
@@ -346,7 +352,7 @@ def _load_json(self, english_aliases, lab_to_townhall):
         id = 2000
         for c, [supercell_name, meta] in enumerate(data.items()):
 
-            # Not interested if it doesn't have a TID, since it likely isn't a real troop.
+            #Not interested if it doesn't have a TID, since it likely isn't a real troop.
```

Comment:
```suggestion
            # Not interested if it doesn't have a TID, since it likely isn't a real troop.
```
I would revert this one